modules/appeals_api/app/swagger/higher_level_reviews/v0/api_description_dev.md

```diff
@@ -35,5 +37,3 @@ Test submissions do not progress through the same statuses as the production env
 
 #### Status Caching
 Due to system limitations, status attribute data for the GET `/forms/200996/{uuid}` endpoint is cached for 1 hour. The updated_at field indicates the last time the status for a given GUID was updated.
-
-### [Terms of service](https://dev-developer.va.gov/terms-of-service)
```

Comment:
The "Terms of service" links in these updates are removed because there's already a ToS link in the OAS JSON's data - these links in the descriptions are duplicative